frontend/main.py

```diff
@@ -131,6 +131,11 @@ def get_hadith(urn: int):
     else:
         return NotFound()
 
+@app.route('/queue_names/', methods=['GET'])
```

Comment:
we can change the path to 'queues' it just so happens that we get names right now only in the response. 
the response should be a list of objects with name property